corral/main.pony

```diff
@@ -35,16 +39,23 @@ actor Main
           CommandSpec.leaf("list",
             "Lists the deps and packages, including corral details.")?
           CommandSpec.leaf("clean",
-            "Cleans up repo cache and working corral. Default is to clean only working corral.", [
-              OptionSpec.bool("all", "Clean both repo cache and working corral." where short' = 'a', default' = false)
-              OptionSpec.bool("repos", "Clean repo cache only." where short' = 'r', default' = false)
+            "Cleans up repo cache and working corral. Default is to clean" +
+            " only working corral.", [
+              OptionSpec.bool("all",
```

Comment:
the OptionSpec calls here are not in accordance with the style guide. The `"all",` should be on the next line. 

Same applies to CommandSpec. calls below.

Reply:
Looks like this whole section needs that cleanup.